src/WebJobs.Extensions.DurableTask/DurabilityProvider.cs

```diff
@@ -82,6 +85,11 @@ public DurabilityProvider(string storageProviderName, IOrchestrationService serv
         /// </summary>
         public virtual TimeSpan LongRunningTimerIntervalLength { get; set; }
 
+        /// <summary>
+        /// Event source name (e.g. DurableTask-AzureStorage).
+        /// </summary>
+        public string EventSourceName => this.eventSourceName;
```

Comment:
Not sure if this is overoptimizing for a situation that might not happen, but maybe instead of a single EventSourcName, what if we supported an IEnumerable<string> of event source names. I don't think any providers currently need more than one, but it's theoretically possible in the future?

Not sold on needing this change.